src/include/binder/binder.h

```diff
@@ -6,6 +6,7 @@
 #include "binder/query/query_graph.h"
 #include "catalog/catalog_entry/table_catalog_entry.h"
 #include "common/copier_config/file_scan_info.h"
+#include "common/enums/column_evaluate_type.h"
```

Comment:
Let's try to maintain smaller header include as it will propagate. Double check if this is needed.

Reply:
This include is needed now due to changes on line 130 of the same file.